src/cascadia/TerminalApp/PowershellCoreProfileGenerator.cpp

```diff
@@ -322,8 +323,17 @@ std::vector<TerminalApp::Profile> PowershellCoreProfileGenerator::GenerateProfil
         // (or the closest approximation thereof). It may choose a preview instance as the "best" if it is a higher version.
         auto firstProfile = profiles.begin();
         firstProfile->SetGuid(PowershellCoreGuid);
-        firstProfile->SetName(L"PowerShell");
+        firstProfile->SetName(POWERSHELL_PREFERRED_PROFILE_NAME);
     }
 
     return profiles;
 }
+
+// Function Description:
+// - Returns the thing it's named for.

```

Comment:
too cheeky?

Reply:
Eh. Always nice to have a few of these sprinkled throughout haha